src/main/scala/outwatch/Sink.scala

```diff
@@ -1,12 +1,14 @@
 package outwatch
 
 import cats.effect.IO
-import rxscalajs.facade.SubjectFacade
-import rxscalajs.subscription.Subscription
-import rxscalajs.{Observable, Observer, Subject}
+import monix.execution.Scheduler.Implicits.global
```

Comment:
Is it safe to import the global execution scheduler everywhere?

Reply:
It is ok, but it makes the [batched execution model](https://monix.io/docs/2x/execution/scheduler.html#execution-model) the default everywhere. It would be better to have the scheduler an implicit argument where it's needed, but that would have been a PR with more changes vs the rxscala version (which essentially doesn't have an  option of picking a different Scheduler).